devicetypes/smartthings/zwave-dimmer-switch-generic.src/zwave-dimmer-switch-generic.groovy

```diff
@@ -24,28 +24,28 @@ metadata {
 
 		fingerprint inClusters: "0x26", deviceJoinName: "Z-Wave Dimmer"
```

Comment:
Are any of the fingerprints in here actually lights? I wonder if we should just change the `ocfDeviceType` at the DTH level to `"oic.d.switch"` and then override the few smart plugs